src/common/attributes/usePositionAttributes.js

```diff
@@ -224,6 +224,10 @@ export default (t) => useMemo(() => ({
     name: t('positionFuelConsumption'),
     type: 'number',
   },
+  fuelUsed: {
```

Comment:
let's move this above `fuelConsumption` to match the order we have in `Position.java`